docs/source/external_tutorials/tutorial_cellpose_segmentation.ipynb

```diff
@@ -0,0 +1,400 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Nuclei segmentation using cellpose"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial shows how we can easily use an external segmentation packages in the *Squidpy* framework."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import squidpy as sq\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare cellpose functionality for *Squidpy*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use the cellpose package, we need to install it with `pip install cellpose`.\n",
```

Comment:
I would move this up to the general explanation (after line 14). Can you also add one sentence explaining what cellpose is? E.g. "cellpose (link-to-code) is a novel anatomical segmentation algorithm. To use it in this example, we need to install it ..."
For referencing the paper, please use `data-cite`, as done in the tangram tutorial: https://squidpy.readthedocs.io/en/latest/external_tutorials/tutorial_tangram.html 
`**Tangram** (<cite data-cite="tangram">Biancalani et al. (2020)</cite>, [code](https://github.com/broadinstitute/Tangram))`. The reference goes in `docs/source/references.bib` (@michalk8 please correct me if im wrong).